pkg/assembler/backends/neo4j/pkg.go

```diff
@@ -406,24 +380,15 @@ func (c *neo4jClient) Packages(ctx context.Context, pkgSpec *model.PkgSpec) ([]*
 
 			for result.Next() {
 				pkgQualifiers := []*model.PackageQualifier{}
-				if pkgSpec.MatchOnlyEmptyQualifiers != nil && !*pkgSpec.MatchOnlyEmptyQualifiers {
-					qualifierNode := result.Record().Values[5].(dbtype.Node)
-					if len(pkgSpec.Qualifiers) > 0 {
-						for _, qualifier := range pkgSpec.Qualifiers {
-							qualifierKey := removeInvalidCharFromProperty(qualifier.Key)
-							pkgQualifier := &model.PackageQualifier{
-								Key:   qualifierKey,
-								Value: qualifierNode.Props[qualifierKey].(string),
-							}
-							pkgQualifiers = append(pkgQualifiers, pkgQualifier)
-						}
-					} else {
-						for key, value := range qualifierNode.Props {
-							pkgQualifier := &model.PackageQualifier{
-								Key:   key,
-								Value: value.(string),
+				if result.Record().Values[5] != nil {
```

Comment:
In #476 I make this into a helper function as it's used in more places. I can rebase and get that added to that PR.

Reply:
Yeah, it's one of the things I'm (was) planning to refactor to utilities.